src/pages/docs/administration/reporting/report-on-deployments-with-sql.md

```diff
@@ -29,9 +28,7 @@ The table is completely denormalized, and should have any information that you m
 
 ## How often is the data updated?
 
-The data in the table (and exposed by the feed) updates every 30 seconds to add deployments that have recently completed.
-
-Since the data is denormalized, changing the name of a project or environment, or changing the version number of a release, may result in stale data. Octopus corrects this every 24 hours on a schedule.
+The data in the table (and exposed by the feed) updates every time any data related to deployments changes. This includes changes such as changing the name of a project or environment, or changing the version number of a release. The data should always be up-to-date within the last 30 seconds, however if Octopus is performing many operations this may increase to several minutes.
```

Comment:
I arbitrarily picked these timings (30 seconds to match the old 30 seconds). Happy to change them if they don't feel accurate.

Reply:
Nit: Or not mention it?
```suggestion
The data in the table (and exposed by the feed) updates every time any data related to deployments changes. This includes changes such as changing the name of a project or environment, or changing the version number of a release. The data should always be up-to-date, however if Octopus is performing many operations, data could be stale up to several minutes.
```